src/main/webapp/WEB-INF/pages/checkoutPage.jsp

```diff
@@ -0,0 +1,126 @@
+<%@ page contentType="text/html;charset=UTF-8" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ taglib prefix="tags" tagdir="/WEB-INF/tags" %>
+<jsp:useBean id="cart" type="com.es.phoneshop.model.cart.Cart" scope="request"/>
+
+<tags:master pageTitle="CheckoutPage">
+    <table>
+        <thead>
+        <tr>
+            <td>
+                Image
+            </td>
+            <td>
+                Description
+            </td>
+            <td>
+                Quantity
+            </td>
+            <td>
+                Price
+            </td>
+        </tr>
+        </thead>
+        <c:forEach var="cartItem" items="${cart.cartItems}">
+            <tr>
+                <td>
+                    <img class="product-tile"
+                         src="https://raw.githubusercontent.com/andrewosipenko/phoneshop-ext-images/master/${cartItem.product.imageUrl}">
+                </td>
+                <td>
+                    <c:url value="/products/${cartItem.product.id}" var="productUrl"/>
+                    <a href="${productUrl}">${cartItem.product.description}</a>
+                </td>
+                <td>
+                        ${cartItem.quantity}
+                </td>
+                <td class="price">
+                    <fmt:formatNumber value="${cartItem.product.price}" type="currency"
+                                      currencySymbol="${cartItem.product.currency.symbol}"/>
+                </td>
+            </tr>
+        </c:forEach>
+        <tr>
+            <td colspan="3" class="price">
+                Price of products:
+            </td>
+            <td>
+                <fmt:formatNumber value="${cart.totalPrice}" type="currency" currencySymbol="USD"/>
+            </td>
+        </tr>
+    </table>
+
+    <form method="post">
+        <p>
+            <label for="firstName">First name</label>
+            <input id="firstName" name="firstName" value="${firstName}"/>
+        </p>
+        <c:if test="${not empty firstNameError}">
+            <p>
+                <span class = "class-color-red">${firstNameError}</span>
```

Comment:
Here and in other places: redundant spaces
class = "class-color-red" -> class="class-color-red"